src/clj/game/cards/identities.clj

```diff
@@ -93,9 +93,11 @@
                   :choices {:max 1
                             :req #(and (= (:side %) "Runner")
                                        (in-hand? %))}
-                  :req (req (> (count (:hand runner)) 0))
+                  :req (req (and (pos? (count (:hand runner)))
+                                 (:runner-phase-12 @state)))
```

Comment:
This might make Apex players sad if they don't remember to click Apex during the 1.2 window. Restricting when this can be used just makes correcting mistakes harder I think.